cmd/cstorvolumeclaim/controller.go

```diff
@@ -385,3 +398,199 @@ func BaseLabels(cvc *apis.CStorVolumeClaim) map[string]string {
 	}
 	return base
 }
+
+// cvcNeedResize returns true if a cvc requests a resize operation.
+func (c *CVCController) cvcNeedResize(cvc *apis.CStorVolumeClaim) bool {
+	// only bound cstorvolumeclaim can be resized
+	if cvc.Status.Phase != apis.CStorVolumeClaimPhaseBound {
+		return false
+	}
+
+	cv, err := c.clientset.OpenebsV1alpha1().CStorVolumes(cvc.Namespace).
+		Get(cvc.Name, metav1.GetOptions{})
+	if err != nil {
+		runtime.HandleError(fmt.Errorf("falied to get cv %s: %v", cvc.Name, err))
+		return false
+	}
+
+	requestCVCSize := cvc.Spec.Capacity[v1.ResourceStorage]
+	actualCVCSize := cvc.Status.Capacity[v1.ResourceStorage]
+	if requestCVCSize.Cmp(actualCVCSize) > 0 {
```

Comment:
I think here we need to have another check 
`if cv.Spec.Capacity.Cmp(cv.Status.Capacity) ==  0 && 	if requestCVCSize.Cmp(actualCVCSize) > 0 {...}`
It will help to stop different multiple resize request at time.

Reply:
As `CV.Status.Capacity` is `0` while create, we can have this check as well